src/Tickets/Order_Modifiers/Custom_Tables/Order_Modifier_Relationships.php

```diff
@@ -102,12 +94,16 @@ protected function after_update( array $results ): array {
 		}
 
 		global $wpdb;
-		$table_name = self::table_name( true );
+		$table_name        = self::table_name();
+		$parent_table_name = Order_Modifiers::table_name();
+		$parent_table_uid  = Order_Modifiers::uid_column();
+		$wp_posts_table    = $wpdb->posts;
 
-		// Check if the table exists first.
-		if ( ! $this->exists() ) {
-			return $results;
-		}
+		// Add foreign key for `modifier_id`.
+		$this->add_foreign_key( $table_name, 'fk_modifier_id', 'modifier_id', $parent_table_name, $parent_table_uid );
+
+		// Add foreign key for `post_id`.
+		$this->add_foreign_key( $table_name, 'fk_post_id', 'post_id', $wp_posts_table, 'ID' );
```

Comment:
Avoid Foreign keys, keep it as soft references. Avoid Delete Cascading too, there is a reason why WordPress doesn't use those, it breaks in tons of servers.